core/src/bitcoin/bech32/Bech32Parameters.cpp

```diff
@@ -73,6 +73,30 @@ namespace ledger {
                             {0x08}
                     };
                     return BITCOIN_CASH;
+                } else if (networkIdentifier == "dgb") {
+                    //https://github.com/digibyte/digibyte/blob/master/src/bech32.cpp
+                    static const Bech32Struct DIGIBYTE = {
+                            "digibyte",
+                            "dgb",
+                            "1",
+                            6,
+                            {0x3b6a57b2ULL, 0x26508e6dULL, 0x1ea119faULL, 0x3d4233ddULL, 0x2a1462b3ULL},
+                            {0x00},
+                            {0x00}
+                    };
+                    return DIGIBYTE;
+                } else if (networkIdentifier == "ltc") {
+                    //https://github.com/litecoin-project/litecoin/blob/master/src/bech32.cpp
+                    static const Bech32Struct LITECOIN = {
+                            "litecoin",
+                            "ltc",
+                            "1",
+                            6,
+                            {0x3b6a57b2ULL, 0x26508e6dULL, 0x1ea119faULL, 0x3d4233ddULL, 0x2a1462b3ULL},
+                            {0x00},
+                            {0x00}
+                    };
+                    return LITECOIN;
```

Comment:
Since the `if/else` statements grow up day after day, we can consider to put all these `Bech32Struct` to a `constexpr std::unordored_map<Bech32Struct>` and then compute a find operation on it.

Reply:
Same, this should be in a dedicated PR because we have multiple places where we do that, so let's open a ticket for this !